src/tests/multihost/alltests/test_sssctl_analyzer.py

```diff
@@ -295,3 +301,35 @@ def test_analyze_parse_child_logs(self, multihost, backupsssdconf):
             assert re.findall(r"RID#[0-9]*] Received error code", stdout)
         else:
             pytest.fail(f"{user} sucessful to login")
+
+    def test_non_root_privileged(self, multihost, backupsssdconf, localusers):
+        """
+        :title: SSSD: `sssctl analyze` command shouldn't require 'root' privileged
+        :id: 51a69d4e-7ae4-11ed-95a5-845cf3eff344
+        :description: `sssctl analyze` command shouldn't require
+            'root' privileged when run with `--logdir`
+            pointing to otherwise accessible files.
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2142960
+                   https://bugzilla.redhat.com/show_bug.cgi?id=2142794
+                   https://bugzilla.redhat.com/show_bug.cgi?id=2142961
+        :steps:
+          1. Create directory
+          2. Copy logs to above directory
+          3. Change ownership to a local user
+          4. Try sssctl analyze command pointing to above
+            directory with root and loacluser
+        :expectedresults:
+          1. Should succeed
+          2. Should succeed
+          3. Should succeed
+          4. Should succeed
+        """
+        execute_cmd(multihost, "mkdir /tmp/sssd; "
+                               "cp -vf /var/log/sssd/* /tmp/sssd;"
+                               " chown user5000 /tmp/sssd/;"
+                               "chown user5000 /tmp/sssd/*")
+        cmd_root = execute_cmd(multihost, "sssctl analyze --logdir /tmp/sssd").stdout_text
+        cmd_user500 = execute_cmd(multihost, "runuser -l user5000 -c "
+                                             "'sssctl analyze --logdir /tmp/sssd'").stdout_text
+        execute_cmd(multihost, "rm -vfr /tmp/sssd")
+        assert cmd_root == cmd_user500
```

Comment:
The assert does not have any error message.
Please add message that will be shown when the test fails.

Reply:
what is the usecase here ? test will fail with AssertionError